flow/src/main/scala/org/alephium/flow/core/FlowUtils.scala

```diff
@@ -346,7 +348,7 @@ trait FlowUtils
     val bestDeps = getBestDeps(groupIndex)
 
     val currentTs = TimeStamp.now()
-    getRelevantUtxos(groupIndex, bestDeps, lockupScript).map { utxos =>
+    getRelevantUtxos(groupIndex, bestDeps, lockupScript, Int.MaxValue).map { utxos =>
```

Comment:
what if they have more than `Int.MaxValue` utxos?

Reply:
we will only read up to `Int.MaxValue` utxos, which is fine. It's not likely to happen in real

Reply:
If that's the case, all the `collection.size` would fail